llm/llama.go

```diff
@@ -119,6 +119,67 @@ type llama struct {
 	api.Options
 }
 
+const (
+	llamaFileTypeF32 FileType = iota
+	llamaFileTypeF16
+	llamaFileTypeQ4_0
+	llamaFileTypeQ4_1
+	llamaFileTypeQ4_1_F16
+	llamaFileTypeQ8_0 = iota + 3
```

Comment:
why leave this gap in the iota?

Reply:
This is parsed from the model binary. T gap, 5 and 6, are mapped to Q4_2 and Q4_3 which have been removed.